profiles/common/modules/custom/ecas/includes/ecas.inc

```diff
@@ -213,7 +213,7 @@ function ecas_login_page() {
 function ecas_logout($full_logout = FALSE) {
   global $user;
 
-  watchdog('user', 'Session closed for %name.', array('%name' => theme('placeholder', $user->name)));
+  watchdog('user', 'Session closed for %name.', array('%name' => format_username($user)));
```

Comment:
After some check and confirmation by @Fefaine, there is no need to have a `check_plain()` here, the watchdog will do it for us.